client/my-sites/site-settings/form-general.jsx

```diff
@@ -442,6 +444,46 @@ const FormGeneral = React.createClass( {
 		);
 	},
 
+	helloVoteOption() {
+		// Note that years and months below are zero indexed
+		const site = this.props.site;
+
+		if ( site.jetpack ) {
+			return null;
+		}
+
+		return (
+			<FormFieldset>
+				<legend>{ this.translate( 'US Voter Registration Form' ) }</legend>
+				<ul>
+					<li>
+						<FormLabel>
```

Comment:
You could actually also attach onClick somewhere higher, so you are not dependant on FormCheckbox events
(they are quirky)

So FormCheckbox would only output current state and you would toggle appropriate action by clickong somewhere higher in the component tree